src/JsonRpc/RequestRouterBase.cs

```diff
@@ -99,7 +99,7 @@ public virtual async Task<ErrorResponse> RouteRequest(TDescriptor descriptor, Re
                     // TODO: Try / catch for Internal Error
                     try
                     {
-                        if (descriptor == default)
+                        if (descriptor.Equals(default))
```

Comment:
@mholo65 @david-driscoll for some reason I needed to make this change even though I didn't touch this code path... not sure why...

Reply:
I reverted this... but I don't know why it's happening and don't think it's related to my change.

Reply:
Problem is still on building in Azure DevOps as it's building with .NET SDK 3.1.100... I'll try to fix this asap.